deals/deals_test.go

```diff
@@ -95,7 +95,7 @@ func storeMultiMiner(m *Module, client *apistruct.FullNodeStruct, numMiners int,
 	for i := 0; i < numMiners; i++ {
 		cfgs[i] = StorageDealConfig{
 			Miner:      miners[i].String(),
-			EpochPrice: 1000000,
+			EpochPrice: 500000000,
```

Comment:
The miner in the devnet is using now the default ask price, which is 500m attofil. 
Unfortunately, I had to remove a call which lowered his price since it has a bug (that I'll fix in Lotus in some days).
To avoid doing that PR, needed to deal with the default price which is pretty high.